_sass/_hca.scss

```diff
@@ -1,4 +1,14 @@
 // TODO: properly incorporate usds web standards for this
+.hca-landing-page {
+  a.va-button-primary {
+    text-decoration: none !important;
+  }
+}
+
+#frequently-asked-questions {
```

Comment:
It looks like `#frequently-asked-question`  was removed from the HTML, so we can remove it from the CSS too.